.github/workflows/build.yml

```diff
@@ -1222,9 +1082,8 @@ jobs:
 
           if [ "$PRERELEASE" == "false" ]
           then
-            # The order here matters, because the Chinese build has a slightly higher version code due to the suffix of 1 vs 0
             ./pico-cli upload-build --app-id $PICO_APP_ID --app-secret $PICO_APP_SECRET --region noncn --apk OpenBrush_Pico_$VERSION.apk --channel 2 --notes-en "Version $VERSION" --device 'PICO Neo3,PICO Neo3 Pro,PICO Neo3 Eye,PICO 4'
-            ./pico-cli upload-build --app-id $PICO_APP_ID --app-secret $PICO_APP_SECRET --region noncn --apk OpenBrush_Pico_CN_$VERSION.apk --channel 1 --notes-en "Version $VERSION" --device 'PICO Neo3,PICO Neo3 Pro,PICO Neo3 Eye,PICO 4'
+            ./pico-cli upload-build --app-id $PICO_APP_ID --app-secret $PICO_APP_SECRET --region cn --apk OpenBrush_Pico_$VERSION.apk --channel 1 --notes-en "Version $VERSION" --device 'PICO Neo3,PICO Neo3 Pro,PICO Neo3 Eye,PICO 4'
```

Comment:
I fear that this will fail; IIRC, we can't publish the same version code to both CN and non-CN.

Reply:
Ah, that's incredibly dumb if so. Can we give it a try? The code I'm using to unify the builds is from Pico's official Unreal documentation, so it implies the same build should be supported?

Reply:
Sure. Whenever we merge this, just be on call for followups with failed publishes, including possibly oculus (because of course they will), which is something we can't easily test

Reply:
Yep, I will also try taking a build and manually uploading to a test app I have before merging

Reply:
ok, just don't upload to our regular build. The way they do channels is a bit weird, but the upshot is that you can mess things up by pushing random android build IDs

Reply:
![image](https://github.com/user-attachments/assets/ba9c9267-d9e5-4db2-a1a7-01999aabf062)

Reply:
I hope that's accurate :-)